reset-pass/account-validator-tool/src/java/org/sakaiproject/accountvalidator/tool/controller/MainController.java

```diff
@@ -0,0 +1,1283 @@
+        
+        // va = getValidationAccount(tokenId, model);
+        // getValidationAccount {
+        
+        if (tokenId == null || "".equals(tokenId)) {
+
+            log.debug("Valid token not found");
+            model.addAttribute("warnMsg", messageSource.getMessage("msg.noCode", null, userLocale));
+            return "error.html";
+        }
+              
+        va = validationLogic.getVaLidationAcountBytoken(tokenId);
+        
+        if (va == null) {
+
+            log.debug("Not a valid validation code");
+            model.addAttribute("warnMsg", messageSource.getMessage("msg.noSuchValidation", new String[]{tokenId}, userLocale));
+            return "error.html";
+        // } checks left:    
+        } else if (!va.getAccountStatus().equals(ValidationAccount.ACCOUNT_STATUS_REQUEST_ACCOUNT)) {
+            // this form is not appropriate, no such validation of the required account status
+            model.addAttribute("warnMsg", messageSource.getMessage("msg.noSuchValidation", new String[]{va.getValidationToken()}, userLocale));
+            return "error.html";
+
+        } else if (ValidationAccount.STATUS_CONFIRMED.equals(va.getStatus())) {
+
+            model.addAttribute("warnMsg", messageSource.getMessage("msg.alreadyValidated", null, userLocale));
+            return "error.html";
+            
+        } else if (ValidationAccount.STATUS_EXPIRED.equals(va.getStatus())) {
+            
+            model.addAttribute("warnMsg",  messageSource.getMessage("msg.alreadyValidated", new String[]{va.getValidationToken()}, userLocale));
+            return "error.html";
+        }
+        
+        User u = null;
+
+        try {  
+            u = userDirectoryService.getUser(va.getUserId());
+            log.debug("user ID: "+u.getEid());
+        
+        } catch (UserNotDefinedException e){
+            log.error("user ID does not exist for ValidationAccount with tokenId: " + va.getValidationToken());
+            model.addAttribute("warnMsg", messageSource.getMessage("validate.userNotDefined", new String[]{serverConfigurationService.getString("ui.service", "Sakai")}, userLocale));
+            return "error.html";
+        }
+        
+        // Fill from:
+        
+        model.addAttribute("eid", u.getEid());
+        
+        // for predefined info
+        model.addAttribute("firstName", u.getFirstName());
+        model.addAttribute("surName", u.getLastName());
+
+        boolean passwordPolicyEnabled = true;//userDirectoryService.getPasswordPolicy() != null;
+        String passwordPolicyEnabledJavaScript = "VALIDATOR.isPasswordPolicyEnabled = " + Boolean.toString(passwordPolicyEnabled) + ";";
+        model.addAttribute("passwordPolicyEnabled", passwordPolicyEnabledJavaScript);
+ 
+            // If we have some terms, get the user to accept them.
+            if (!"".equals(serverConfigurationService.getString("account-validator.terms"))) {
```

Comment:
Tabs vs spaces.

Use a variable and StringUtils.
String accountValidatorTerms = serverConfigurationService.getString("account-validator.terms");
if (StringUtils.isNotBlank(accountValidatorTerms)) .....